### ISSUE TITLE
Wrong function call jump during debugging

### ISSUE THREAD

**Issue by damms005** [wallabyjs/public#3136]
### Issue description or question
When debugging, clicking to Step Into a function jumps to a wrong function. See screencast below:

You can see that when the debugging gets to `test/jest/__tests__/bible-study/card-advancement.spec.ts#43` at the call to `addManyCards()`, Stepping Into the function call jumps to "arbitrary?" `getKjvBibleFromLocalOrRemote()` at `src/repositories/bible-repo.ts#10` instead of where `addManyCards()` is defined; which is at `test/jest/__tests__/fixtures/card-utils.ts#94`

https://user-images.githubusercontent.com/9839355/209979264-ac98aa79-57e9-4dbb-bdb5-aa001276ba2b.mp4

**Note**
If it helps, I made the repo public here at https://github.com/damms005/corazon-quasar

### Wallaby diagnostics report

```
{
  editorVersion: '1.74.2',
  pluginVersion: '1.0.348',
  editorType: 'VSCode',
  osVersion: 'linux 5.15.0-56-generic',
  nodeVersion: 'v18.9.0',
  coreVersion: '1.0.1363',
  checksum: 'MGY4OTMzM2Q5ZDZjOTlhYzU3YTI4NGU3MjRkNTc3NWQsMTcwMDk1NjgwMDAwMCww',
  config: {
    diagnostics: {
      jest: {
        config: {
          configs: [
            {
              automock: false,
              cache: true,
              cacheDirectory: '/tmp/jest_rs',
              clearMocks: false,
              coveragePathIgnorePatterns: [ '/node_modules/', '.d.ts$' ],
              cwd: '<homeDir>/Documents/node-apps/quasar-apps/corazon-quasar',
              dependencyExtractor: undefined,
              detectLeaks: false,
              detectOpenHandles: false,
              displayName: undefined,
              errorOnDeprecated: false,
              extensionsToTreatAsEsm: [],
              fakeTimers: { enableGlobally: false },
              filter: undefined,
              forceCoverageMatch: [],
              globalSetup: undefined,
              globalTeardown: undefined,
              globals: { __DEV__: true, watchPathIgnorePatterns: [ '<rootDir>/test/jest/__tests__/fixtures/test-db.sqlite', '<rootDir>/CORAZON_BIBLE_APP/', '<rootDir>/test/cypress/' ] },
              haste: { computeSha1: false, enableSymlinks: false, forceNodeFilesystemAPI: true, throwOnModuleCollision: false },
              id: '94d48f3818bf82742dd86ec94e6849d2',
              injectGlobals: true,
              moduleDirectories: [ 'node_modules' ],
              moduleFileExtensions: [ 'vue', 'js', 'jsx', 'json', 'ts', 'tsx' ],
              moduleNameMapper: [
                [ '^quasar$', 'quasar/dist/quasar.esm.prod.js' ],
                [ '^~/(.*)$', '<homeDir>/Documents/node-apps/quasar-apps/corazon-quasar/$1' ],
                [ '^src/(.*)$', '<homeDir>/Documents/node-apps/quasar-apps/corazon-quasar/src/$1' ],
                [ '^app/(.*)$', '<homeDir>/Documents/node-apps/quasar-apps/corazon-quasar/$1' ],
                [ '^components/(.*)$', '<homeDir>/Documents/node-apps/quasar-apps/corazon-quasar/src/components/$1' ],
                [ '^layouts/(.*)$', '<homeDir>/Documents/node-apps/quasar-apps/corazon-quasar/src/layouts/$1' ],
                [ '^pages/(.*)$', '<homeDir>/Documents/node-apps/quasar-apps/corazon-quasar/src/pages/$1' ],
                [ '^assets/(.*)$', '<homeDir>/Documents/node-apps/quasar-apps/corazon-quasar/src/assets/$1' ],
                [ '^boot/(.*)$', '<homeDir>/Documents/node-apps/quasar-apps/corazon-quasar/src/boot/$1' ],
                [ '.*css$', '@quasar/quasar-app-extension-testing-unit-jest/stub.css' ]
              ],
              modulePathIgnorePatterns: [],
              modulePaths: undefined,
              prettierPath: 'prettier',
              resetMocks: false,
              resetModules: false,
              resolver: undefined,
              restoreMocks: false,
              rootDir: '<homeDir>/Documents/node-apps/quasar-apps/corazon-quasar',
              roots: [ '<homeDir>/Documents/node-apps/quasar-apps/corazon-quasar' ],
              runner: '<homeDir>/Documents/node-apps/quasar-apps/corazon-quasar/node_modules/jest-runner/build/index.js',
              runtime: undefined,
              sandboxInjectedGlobals: [],
              setupFiles: [],
              setupFilesAfterEnv: [],
              skipFilter: false,
              skipNodeResolution: undefined,
              slowTestThreshold: 5,
              snapshotFormat: { escapeString: false, printBasicPrototype: false },
              snapshotResolver: undefined,
              snapshotSerializers: [],
              testEnvironment: '<homeDir>/Documents/node-apps/quasar-apps/corazon-quasar/node_modules/jest-environment-node/build/index.js',
              testEnvironmentOptions: {},
              testLocationInResults: false,
              testMatch: [
                '<homeDir>/Documents/node-apps/quasar-apps/corazon-quasar/test/jest/__tests__/**/*.(spec|test).+(ts|js)?(x)',
                '<homeDir>/Documents/node-apps/quasar-apps/corazon-quasar/src/**/*.jest.(spec|test).+(ts|js)?(x)'
              ],
              testPathIgnorePatterns: [ '/node_modules/' ],
              testRegex: [],
              testRunner: '<homeDir>/Documents/node-apps/quasar-apps/corazon-quasar/node_modules/jest-circus/runner.js',
              transform: [
                [ '^(quasar|quasar/lang|lodash-es).+\\.js$', '<homeDir>/Documents/node-apps/quasar-apps/corazon-quasar/node_modules/babel-jest/build/index.js', {} ],
                [ '^.+\\.(ts|js|html)$', '<homeDir>/Documents/node-apps/quasar-apps/corazon-quasar/node_modules/ts-jest/dist/index.js', {} ]
              ],
              transformIgnorePatterns: [ 'node_modules/(?!(quasar|quasar/lang|lodash-es))' ],
              unmockedModulePathPatterns: undefined,
              watchPathIgnorePatterns: [
                '<homeDir>/Documents/node-apps/quasar-apps/corazon-quasar/test/jest/__tests__/fixtures/test-db.sqlite',
                '<homeDir>/Documents/node-apps/quasar-apps/corazon-quasar/CORAZON_BIBLE_APP/',
                '<homeDir>/Documents/node-apps/quasar-apps/corazon-quasar/test/cypress/'
              ]
            }
          ],
          globalConfig: {
            bail: 0,
            changedFilesWithAncestor: false,
            changedSince: undefined,
            ci: false,
            collectCoverage: false,
            collectCoverageFrom: [ 'src/**/*.vue', 'src/**/*.js', 'src/**/*.ts', 'src/**/*.jsx', 'src/**/*.tsx' ],
            coverageDirectory: '<homeDir>/Documents/node-apps/quasar-apps/corazon-quasar/test/jest/coverage',
            coverageProvider: 'babel',
            coverageReporters: [ 'json', 'text', 'lcov', 'clover' ],
            coverageThreshold: { global: {} },
            detectLeaks: false,
            detectOpenHandles: false,
            errorOnDeprecated: false,
            expand: false,
            filter: undefined,
            findRelatedTests: false,
            forceExit: false,
            globalSetup: undefined,
            globalTeardown: undefined,
            json: false,
            lastCommit: false,
            listTests: false,
            logHeapUsage: false,
            maxConcurrency: 5,
            maxWorkers: 7,
            noSCM: undefined,
            noStackTrace: false,
            nonFlagArgs: undefined,
            notify: false,
            notifyMode: 'failure-change',
            onlyChanged: false,
            onlyFailures: false,
            outputFile: undefined,
            passWithNoTests: false,
            projects: [],
            replname: undefined,
            reporters: undefined,
            rootDir: '<homeDir>/Documents/node-apps/quasar-apps/corazon-quasar',
            runTestsByPath: false,
            seed: -1022326968,
            shard: undefined,
            showSeed: undefined,
            silent: undefined,
            skipFilter: false,
            snapshotFormat: { escapeString: false, printBasicPrototype: false },
            testFailureExitCode: 1,
            testNamePattern: undefined,
            testPathPattern: '',
            testResultsProcessor: undefined,
            testSequencer: '<homeDir>/Documents/node-apps/quasar-apps/corazon-quasar/node_modules/@jest/test-sequencer/build/index.js',
            testTimeout: undefined,
            updateSnapshot: 'new',
            useStderr: false,
            verbose: undefined,
            watch: false,
            watchAll: false,
            watchPlugins: undefined,
            watchman: true,
            workerIdleMemoryLimit: undefined
          },
          hasDeprecationWarnings: false,
          wallaby: {
            roots: [],
            watchPathIgnorePatterns: [
              '/node_modules/',
              '\\./dist/|\\./build/|\\./coverage/|\\./git/|/\\..+/',
              '/tmp/jest_rs',
              '\\./test/jest/__tests__/fixtures/test-db.sqlite',
              '\\./CORAZON_BIBLE_APP/',
              '\\./test/cypress/',
              '\\./test/jest/coverage'
            ],
            testPathIgnorePatterns: [
              '/node_modules/',
              '\\./dist/|\\./build/|\\./coverage/|\\./git/|/\\..+/',
              '/tmp/jest_rs',
              '\\./test/jest/__tests__/fixtures/test-db.sqlite',
              '\\./CORAZON_BIBLE_APP/',
              '\\./test/cypress/',
              '\\./test/jest/coverage'
            ],
            testMatch: [ 'test/jest/__tests__/**/*.(spec|test).+(ts|js)?(x)', 'src/**/*.jest.(spec|test).+(ts|js)?(x)' ],
            testRegex: []
          }
        }
      }
    },
    testFramework: { version: 'jest@24.8.0', configurator: 'jest@24.8.0', reporter: 'jest@24.8.0', starter: 'jest@24.8.0', autoDetected: true },
    filesWithCoverageCalculated: [ 'src/**/*.vue', 'src/**/*.js', 'src/**/*.ts', 'src/**/*.jsx', 'src/**/*.tsx' ],
    filesWithNoCoverageCalculated: [ /.d.ts$/ ],
    globalSetup: false,
    dot: true,
    files: [
      { pattern: '/node_modules/', regexp: /\/node_modules\//, ignore: true, trigger: true, load: true, file: true, test: true },
      {
        pattern: '\\./dist/|\\./build/|\\./coverage/|\\./git/|/\\..+/',
        regexp: /\.\/dist\/|\.\/build\/|\.\/coverage\/|\.\/git\/|\/\..+\//,
        ignore: true,
        trigger: true,
        load: true,
        file: true,
        test: true
      },
      { pattern: '/tmp/jest_rs', regexp: /\/tmp\/jest_rs/, ignore: true, trigger: true, load: true, file: true, test: true },
      {
        pattern: '\\./test/jest/__tests__/fixtures/test-db.sqlite',
        regexp: /\.\/test\/jest\/__tests__\/fixtures\/test-db.sqlite/,
        ignore: true,
        trigger: true,
        load: true,
        file: true,
        test: true
      },
      { pattern: '\\./CORAZON_BIBLE_APP/', regexp: /\.\/CORAZON_BIBLE_APP\//, ignore: true, trigger: true, load: true, file: true, test: true },
      { pattern: '\\./test/cypress/', regexp: /\.\/test\/cypress\//, ignore: true, trigger: true, load: true, file: true, test: true },
      { pattern: '\\./test/jest/coverage', regexp: /\.\/test\/jest\/coverage/, ignore: true, trigger: true, load: true, file: true, test: true },
      { pattern: '**/**', ignore: false, trigger: true, load: true, order: 1 },
      { pattern: 'test/jest/__tests__/**/*.(spec|test).+(ts|js)?(x)', ignore: true, trigger: true, load: true, file: true },
      { pattern: 'src/**/*.jest.(spec|test).+(ts|js)?(x)', ignore: true, trigger: true, load: true, file: true }
    ],
    tests: [
      { pattern: '/node_modules/', regexp: /\/node_modules\//, ignore: true, trigger: true, load: true, test: true, file: false },
      {
        pattern: '\\./dist/|\\./build/|\\./coverage/|\\./git/|/\\..+/',
        regexp: /\.\/dist\/|\.\/build\/|\.\/coverage\/|\.\/git\/|\/\..+\//,
        ignore: true,
        trigger: true,
        load: true,
        test: true,
        file: false
      },
      { pattern: '/tmp/jest_rs', regexp: /\/tmp\/jest_rs/, ignore: true, trigger: true, load: true, test: true, file: false },
      {
        pattern: '\\./test/jest/__tests__/fixtures/test-db.sqlite',
        regexp: /\.\/test\/jest\/__tests__\/fixtures\/test-db.sqlite/,
        ignore: true,
        trigger: true,
        load: true,
        test: true,
        file: false
      },
      { pattern: '\\./CORAZON_BIBLE_APP/', regexp: /\.\/CORAZON_BIBLE_APP\//, ignore: true, trigger: true, load: true, test: true, file: false },
      { pattern: '\\./test/cypress/', regexp: /\.\/test\/cypress\//, ignore: true, trigger: true, load: true, test: true, file: false },
      { pattern: '\\./test/jest/coverage', regexp: /\.\/test\/jest\/coverage/, ignore: true, trigger: true, load: true, test: true, file: false },
      { pattern: 'test/jest/__tests__/**/*.(spec|test).+(ts|js)?(x)', ignore: false, trigger: true, load: true, test: true, order: 2 },
      { pattern: 'src/**/*.jest.(spec|test).+(ts|js)?(x)', ignore: false, trigger: true, load: true, test: true, order: 3 }
    ],
    runAllTestsInAffectedTestFile: false,
    updateNoMoreThanOneSnapshotPerTestFileRun: false,
    compilers: {},
    logLimits: { inline: { depth: 5, elements: 5000 }, values: { default: { stringLength: 200 }, autoExpand: { elements: 5000, stringLength: 8192, depth: 10 } } },
    preprocessors: {},
    maxConsoleMessagesPerTest: 100,
    autoConsoleLog: true,
    delays: { run: 0, edit: 100, update: 0 },
    workers: { initial: 0, regular: 0, recycle: false },
    teardown: undefined,
    hints: {
      ignoreCoverage: '__REGEXP /ignore coverage|istanbul ignore/',
      ignoreCoverageForFile: '__REGEXP /ignore file coverage/',
      commentAutoLog: '?',
      testFileSelection: { include: '__REGEXP /file\\.only/', exclude: '__REGEXP /file\\.skip/' }
    },
    automaticTestFileSelection: true,
    runSelectedTestsOnly: false,
    mapConsoleMessagesStackTrace: false,
    extensions: {},
    env: { type: 'node', params: {}, runner: '<homeDir>/.nvm/versions/node/v18.9.0/bin/node', viewportSize: { width: 800, height: 600 }, options: { width: 800, height: 600 }, bundle: true },
    reportUnhandledPromises: true,
    slowTestThreshold: 75,
    lowCoverageThreshold: 80,
    runAllTestsWhenNoAffectedTests: false,
    configCode: 'auto.detect#-1443773592'
  },
  packageJSON: {
    dependencies: {
      '@quasar/extras': '^1.0.0',
      '@types/pouchdb': '^6.4.0',
      axios: '^0.21.1',
      'core-js': '^3.6.5',
      filer: '^1.2.0',
      firebase: '^9.6.6',
      firebaseui: '^6.0.0',
      localforage: '^1.10.0',
      lodash: '^4.17.20',
      luxon: '^1.25.0',
      'node-ray': '^1.19.4',
      pouchdb: '^7.3.1',
      quasar: '^2.0.0',
      'register-service-worker': '^1.7.1',
      sweetalert2: '^10.15.5',
      vue: '^3.0.0',
      'vue-router': '^4.0.0'
    },
    devDependencies: {
      '@babel/eslint-parser': '^7.13.14',
      '@faker-js/faker': '^7.5.0',
      '@quasar/app': '^3.0.0',
      '@types/jest': '^29.2.4',
      '@types/lodash': '^4.14.167',
      '@types/luxon': '^1.25.0',
      '@types/node': '^12.20.21',
      '@typescript-eslint/eslint-plugin': '^5.10.0',
      '@typescript-eslint/parser': '^5.10.0',
      eslint: '^8.23.1',
      'eslint-config-prettier': '^8.1.0',
      'eslint-plugin-vue': '^7.0.0',
      'eslint-watch': '^8.0.0',
      jest: '^29.3.1',
      prettier: '^2.5.1',
      sqlite: '^4.1.2',
      sqlite3: '^5.1.0',
      tailwindcss: '^3.0.18',
      'ts-jest': '^29.0.3',
      typescript: '^4.9.4'
    }
  },
  fs: { numberOfFiles: 94 },
  debug: [
    '2022-12-29T15:47:36.518Z config Attempting automatic configuration for angular\n',
    '2022-12-29T15:47:36.521Z angular/cli config Angular CLI not found.\n',
    '2022-12-29T15:47:36.521Z config Finished attempting automatic configuration for angular (3ms)\n',
    '2022-12-29T15:47:36.521Z config Attempting automatic configuration for jest\n',
    '2022-12-29T15:47:36.612Z jest/config Detected Jest.\n',
    '2022-12-29T15:47:36.613Z jest/config Configured Jest.\n',
    '2022-12-29T15:47:36.613Z config Finished attempting automatic configuration for jest (92ms)\n',
    '2022-12-29T15:47:36.614Z project Wallaby Node version: v18.9.0\n',
    '2022-12-29T15:47:36.614Z project Wallaby config: <homeDir>/Documents/node-apps/quasar-apps/corazon-quasar/auto.detect\n',
    '2022-12-29T15:47:36.618Z fs File system starting\n',
    '2022-12-29T15:47:36.657Z fs File system scan completed\n',
    '2022-12-29T15:47:36.665Z project File cache: <homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/projects/640d2a5c13112d34\n',
    '2022-12-29T15:47:36.704Z uiService Listening port 51235\n',
    '2022-12-29T15:47:36.721Z workers Parallelism for initial run: 6, for regular run: 3\n',
    '2022-12-29T15:47:36.721Z workers Starting run worker instance #0\n',
    '2022-12-29T15:47:36.721Z workers Starting run worker instance #1\n',
    '2022-12-29T15:47:36.721Z workers Starting run worker instance #2\n',
    '2022-12-29T15:47:36.721Z workers Starting run worker instance #3\n',
    '2022-12-29T15:47:36.721Z workers Starting run worker instance #4\n',
    '2022-12-29T15:47:36.721Z workers Starting run worker instance #5\n',
    '2022-12-29T15:47:36.722Z workers Web server is listening at 34157\n',
    '2022-12-29T15:47:36.755Z project Stopping process pool\n',
    '2022-12-29T15:47:36.755Z project File cache is up-to-date, starting full test run\n',
    '2022-12-29T15:47:36.758Z project Test run started; run priority: 3\n',
    '2022-12-29T15:47:36.759Z project Running all tests\n',
    '2022-12-29T15:47:36.762Z workers Starting test run, priority: 3\n',
    '2022-12-29T15:47:36.762Z workers Distributing tests between 6 workers\n',
    '2022-12-29T15:47:36.763Z workers Running tests in parallel\n',
    '2022-12-29T15:47:36.763Z nodeRunner Starting sandbox [worker #0, session #4rfvg]\n',
    '2022-12-29T15:47:36.763Z nodeRunner Starting sandbox [worker #1, session #s2d0i]\n',
    '2022-12-29T15:47:36.763Z nodeRunner Starting sandbox [worker #2, session #l4f8m]\n',
    '2022-12-29T15:47:36.763Z nodeRunner Starting sandbox [worker #3, session #ml6h1]\n',
    '2022-12-29T15:47:36.763Z nodeRunner Starting sandbox [worker #4, session #z3thb]\n',
    '2022-12-29T15:47:36.763Z nodeRunner Starting sandbox [worker #5, session #o492j]\n',
    '2022-12-29T15:47:36.764Z nodeRunner Preparing sandbox [worker #0, session #4rfvg]\n',
    '2022-12-29T15:47:36.764Z nodeRunner Preparing sandbox [worker #1, session #s2d0i]\n',
    '2022-12-29T15:47:36.764Z nodeRunner Preparing sandbox [worker #2, session #l4f8m]\n',
    '2022-12-29T15:47:36.764Z nodeRunner Preparing sandbox [worker #3, session #ml6h1]\n',
    '2022-12-29T15:47:36.764Z nodeRunner Preparing sandbox [worker #4, session #z3thb]\n',
    '2022-12-29T15:47:36.764Z nodeRunner Preparing sandbox [worker #5, session #o492j]\n',
    '2022-12-29T15:47:36.906Z workers Started run worker instance (delayed) #2\n',
    '2022-12-29T15:47:36.907Z nodeRunner Prepared sandbox [worker #2, session #l4f8m]\n',
    '2022-12-29T15:47:36.907Z workers [worker #2, session #l4f8m] Running tests in sandbox\n',
    '2022-12-29T15:47:36.909Z workers Started run worker instance (delayed) #0\n',
    '2022-12-29T15:47:36.909Z nodeRunner Prepared sandbox [worker #0, session #4rfvg]\n',
    '2022-12-29T15:47:36.909Z workers [worker #0, session #4rfvg] Running tests in sandbox\n',
    '2022-12-29T15:47:36.923Z workers Started run worker instance (delayed) #3\n',
    '2022-12-29T15:47:36.923Z nodeRunner Prepared sandbox [worker #3, session #ml6h1]\n',
    '2022-12-29T15:47:36.923Z workers [worker #3, session #ml6h1] Running tests in sandbox\n',
    '2022-12-29T15:47:36.926Z workers Started run worker instance (delayed) #1\n',
    '2022-12-29T15:47:36.926Z nodeRunner Prepared sandbox [worker #1, session #s2d0i]\n',
    '2022-12-29T15:47:36.926Z workers [worker #1, session #s2d0i] Running tests in sandbox\n',
    '2022-12-29T15:47:36.935Z workers Started run worker instance (delayed) #5\n',
    '2022-12-29T15:47:36.935Z nodeRunner Prepared sandbox [worker #5, session #o492j]\n',
    '2022-12-29T15:47:36.935Z workers [worker #5, session #o492j] Running tests in sandbox\n',
    '2022-12-29T15:47:36.935Z workers Started run worker instance (delayed) #4\n',
    '2022-12-29T15:47:36.935Z nodeRunner Prepared sandbox [worker #4, session #z3thb]\n',
    '2022-12-29T15:47:36.935Z workers [worker #4, session #z3thb] Running tests in sandbox\n',
    '2022-12-29T15:47:46.060Z workers Scheduling Jest Test Run (l4f8m): 2022-12-29T15:47:37.182Z\n',
    '2022-12-29T15:47:46.062Z workers [l4f8m] Loaded unknown number of test(s)\n',
    '2022-12-29T15:47:46.116Z workers Scheduling Jest Test Run (s2d0i): 2022-12-29T15:47:37.183Z\n',
    '2022-12-29T15:47:46.117Z workers [s2d0i] Loaded unknown number of test(s)\n',
    '2022-12-29T15:47:46.141Z workers Scheduling Jest Test Run (ml6h1): 2022-12-29T15:47:37.182Z\n',
    '2022-12-29T15:47:46.144Z workers [ml6h1] Loaded unknown number of test(s)\n',
    '2022-12-29T15:47:46.144Z workers [ml6h1] Test executed: hashes cards\n',
    '2022-12-29T15:47:46.156Z workers [ml6h1] Run 1 test(s), skipped 0 test(s)\n',
    '2022-12-29T15:47:46.157Z workers Jest Test Run Complete (ml6h1): 2022-12-29T15:47:46.139Z\n',
    '2022-12-29T15:47:46.158Z workers [ml6h1] Sandbox is responsive, closing it\n',
    '2022-12-29T15:47:46.183Z workers Scheduling Jest Test Run (o492j): 2022-12-29T15:47:37.172Z\n',
    '2022-12-29T15:47:46.189Z workers [o492j] Loaded unknown number of test(s)\n',
    '2022-12-29T15:47:46.191Z workers [o492j] Test executed: removes repeated cards from a collection of cards\n',
    '2022-12-29T15:47:46.191Z workers [o492j] Test executed: deletes cards that are out of sync\n',
    '2022-12-29T15:47:46.191Z workers [o492j] Test executed: does not delete anyhow\n',
    '2022-12-29T15:47:46.196Z workers [o492j] Run 3 test(s), skipped 0 test(s)\n',
    '2022-12-29T15:47:46.197Z workers Jest Test Run Complete (o492j): 2022-12-29T15:47:46.181Z\n',
    '2022-12-29T15:47:46.198Z workers [o492j] Sandbox is responsive, closing it\n',
    '2022-12-29T15:47:46.204Z workers [l4f8m] Test executed: can add new card to daily bucket\n',
    '2022-12-29T15:47:46.209Z workers [l4f8m] Run 1 test(s), skipped 0 test(s)\n',
    '2022-12-29T15:47:46.209Z workers Jest Test Run Complete (l4f8m): 2022-12-29T15:47:46.203Z\n',
    '2022-12-29T15:47:46.210Z workers [l4f8m] Sandbox is responsive, closing it\n',
    '2022-12-29T15:47:46.222Z workers Scheduling Jest Test Run (z3thb): 2022-12-29T15:47:37.195Z\n',
    '2022-12-29T15:47:46.223Z workers [z3thb] Loaded unknown number of test(s)\n',
    '2022-12-29T15:47:46.223Z workers [z3thb] Test executed: correctly caches deleted cards\n',
    '2022-12-29T15:47:46.229Z workers [z3thb] Run 1 test(s), skipped 0 test(s)\n',
    '2022-12-29T15:47:46.230Z workers Jest Test Run Complete (z3thb): 2022-12-29T15:47:46.220Z\n',
    '2022-12-29T15:47:46.230Z workers [z3thb] Sandbox is responsive, closing it\n',
    '2022-12-29T15:47:46.234Z workers Scheduling Jest Test Run (4rfvg): 2022-12-29T15:47:37.199Z\n',
    '2022-12-29T15:47:46.240Z workers [4rfvg] Loaded unknown number of test(s)\n',
    '2022-12-29T15:47:46.307Z workers [4rfvg] Test executed: gets references with chapter and start verse only\n',
    '2022-12-29T15:47:46.332Z workers [s2d0i] Test executed: when daily bucket card is advanced, even/odd bucket receives same card\n',
    '2022-12-29T15:47:46.339Z workers [s2d0i] Run 1 test(s), skipped 0 test(s)\n',
    '2022-12-29T15:47:46.339Z workers Jest Test Run Complete (s2d0i): 2022-12-29T15:47:46.331Z\n',
    '2022-12-29T15:47:46.340Z workers [s2d0i] Sandbox is responsive, closing it\n',
    '2022-12-29T15:47:46.360Z workers [4rfvg] Test executed: gets references with ordered chapter and start verse only\n',
    '2022-12-29T15:47:46.363Z workers [4rfvg] Run 2 test(s), skipped 0 test(s)\n',
    '2022-12-29T15:47:46.363Z workers Jest Test Run Complete (4rfvg): 2022-12-29T15:47:46.359Z\n',
    '2022-12-29T15:47:46.364Z workers [4rfvg] Sandbox is responsive, closing it\n',
    '2022-12-29T15:47:46.364Z workers Merging parallel test run results\n',
    '2022-12-29T15:47:46.367Z project Test run finished\n',
    '2022-12-29T15:47:46.368Z project Processed console.log entries\n',
    '2022-12-29T15:47:46.368Z project Processed loading sequences\n',
    '2022-12-29T15:47:46.369Z project Processed executed tests\n',
    '2022-12-29T15:47:46.372Z project Processed code coverage\n',
    '2022-12-29T15:47:46.396Z project Test run result processed and sent to IDE\n',
    '2022-12-29T15:48:53.736Z project Requested to run some tests\n',
    '2022-12-29T15:48:53.736Z project Debugger start test run\n',
    '2022-12-29T15:48:53.738Z project Test run started; run priority: 1\n',
    '2022-12-29T15:48:53.739Z testTask Test files from affected: 0, from deleted or manually requested: 1, from recently changed: 0, from loaded by: 0, from failing: 0\n',
    '2022-12-29T15:48:53.742Z workers Starting test run, priority: 1\n',
    '2022-12-29T15:48:53.742Z nodeRunner Starting sandbox [worker #0, session #50a7t]\n',
    '2022-12-29T15:48:53.742Z nodeRunner Preparing sandbox [worker #0, session #50a7t]\n',
    '2022-12-29T15:48:53.742Z nodeRunner Prepared sandbox [worker #0, session #50a7t]\n',
    '2022-12-29T15:48:53.742Z workers [worker #0, session #50a7t] Running tests in sandbox\n',
    '2022-12-29T15:48:53.927Z workers Scheduling Jest Test Run (50a7t): 2022-12-29T15:48:53.746Z\n',
    '2022-12-29T15:48:53.927Z workers [50a7t] Loaded unknown number of test(s)\n',
    '2022-12-29T15:48:54.100Z workers [50a7t] Test executed: when daily bucket card is advanced, even/odd bucket receives same card\n',
    '2022-12-29T15:48:54.109Z workers [50a7t] Run 1 test(s), skipped 0 test(s)\n',
    '2022-12-29T15:48:54.110Z workers Jest Test Run Complete (50a7t): 2022-12-29T15:48:54.099Z\n',
    '2022-12-29T15:48:54.113Z workers [50a7t] Sandbox is responsive, closing it\n',
    '2022-12-29T15:48:54.114Z project Test run finished\n',
    '2022-12-29T15:48:54.114Z project Processed console.log entries\n',
    '2022-12-29T15:48:54.114Z project Processed loading sequences\n',
    '2022-12-29T15:48:54.114Z project Processed executed tests\n',
    '2022-12-29T15:48:54.118Z project Processed code coverage\n',
    '2022-12-29T15:48:54.119Z project Processed code trace\n',
    '2022-12-29T15:48:54.160Z project Test run result processed and sent to IDE\n',
    '2022-12-29T15:49:12.384Z project Debugger stopped\n',
    '2022-12-29T15:49:22.589Z project Requested to run some tests\n',
    '2022-12-29T15:49:22.589Z project Debugger start test run\n',
    '2022-12-29T15:49:22.591Z project Test run started; run priority: 1\n',
    '2022-12-29T15:49:22.591Z testTask Test files from affected: 0, from deleted or manually requested: 1, from recently changed: 0, from loaded by: 0, from failing: 0\n',
    '2022-12-29T15:49:22.594Z workers Starting test run, priority: 1\n',
    '2022-12-29T15:49:22.594Z nodeRunner Starting sandbox [worker #0, session #dmc24]\n',
    '2022-12-29T15:49:22.594Z nodeRunner Preparing sandbox [worker #0, session #dmc24]\n',
    '2022-12-29T15:49:22.594Z nodeRunner Prepared sandbox [worker #0, session #dmc24]\n',
    '2022-12-29T15:49:22.594Z workers [worker #0, session #dmc24] Running tests in sandbox\n',
    '2022-12-29T15:49:22.759Z workers Scheduling Jest Test Run (dmc24): 2022-12-29T15:49:22.597Z\n',
    '2022-12-29T15:49:22.760Z workers [dmc24] Loaded unknown number of test(s)\n',
    '2022-12-29T15:49:22.940Z workers [dmc24] Test executed: when daily bucket card is advanced, even/odd bucket receives same card\n',
    '2022-12-29T15:49:22.943Z workers [dmc24] Run 1 test(s), skipped 0 test(s)\n',
    '2022-12-29T15:49:22.943Z workers Jest Test Run Complete (dmc24): 2022-12-29T15:49:22.939Z\n',
    '2022-12-29T15:49:22.944Z workers [dmc24] Sandbox is responsive, closing it\n',
    '2022-12-29T15:49:22.944Z project Test run finished\n',
    '2022-12-29T15:49:22.944Z project Processed console.log entries\n',
    '2022-12-29T15:49:22.944Z project Processed loading sequences\n',
    '2022-12-29T15:49:22.944Z project Processed executed tests\n',
    '2022-12-29T15:49:22.946Z project Processed code coverage\n',
    '2022-12-29T15:49:22.946Z project Processed code trace\n',
    '2022-12-29T15:49:22.952Z project Test run result processed and sent to IDE\n',
    '2022-12-29T15:49:42.547Z project Debugger stopped\n',
    '2022-12-29T16:00:45.168Z project Requested to run some tests\n',
    '2022-12-29T16:00:45.168Z project Debugger start test run\n',
    '2022-12-29T16:00:45.169Z project Test run started; run priority: 1\n',
    '2022-12-29T16:00:45.169Z testTask Test files from affected: 0, from deleted or manually requested: 1, from recently changed: 0, from loaded by: 0, from failing: 0\n',
    '2022-12-29T16:00:45.171Z workers Starting test run, priority: 1\n',
    '2022-12-29T16:00:45.171Z nodeRunner Starting sandbox [worker #0, session #rhiw4]\n',
    '2022-12-29T16:00:45.171Z nodeRunner Preparing sandbox [worker #0, session #rhiw4]\n',
    '2022-12-29T16:00:45.171Z nodeRunner Prepared sandbox [worker #0, session #rhiw4]\n',
    '2022-12-29T16:00:45.171Z workers [worker #0, session #rhiw4] Running tests in sandbox\n',
    '2022-12-29T16:00:45.394Z workers Scheduling Jest Test Run (rhiw4): 2022-12-29T16:00:45.174Z\n',
    '2022-12-29T16:00:45.394Z workers [rhiw4] Loaded unknown number of test(s)\n',
    '2022-12-29T16:00:45.603Z workers [rhiw4] Test executed: when daily bucket card is advanced, even/odd bucket receives same card\n',
    '2022-12-29T16:00:45.607Z workers [rhiw4] Run 1 test(s), skipped 0 test(s)\n',
    '2022-12-29T16:00:45.607Z workers Jest Test Run Complete (rhiw4): 2022-12-29T16:00:45.603Z\n',
    '2022-12-29T16:00:45.608Z workers [rhiw4] Sandbox is responsive, closing it\n',
    '2022-12-29T16:00:45.608Z project Test run finished\n',
    '2022-12-29T16:00:45.608Z project Processed console.log entries\n',
    '2022-12-29T16:00:45.608Z project Processed loading sequences\n',
    '2022-12-29T16:00:45.608Z project Processed executed tests\n',
    '2022-12-29T16:00:45.609Z project Processed code coverage\n',
    '2022-12-29T16:00:45.610Z project Processed code trace\n',
    '2022-12-29T16:00:45.619Z project Test run result processed and sent to IDE\n',
    '2022-12-29T16:02:02.804Z project Debugger stopped\n'
  ]
}
```

**Comment by ArtemGovorov** [comment]
Thanks for providing the sample repo.

Wallaby debugger can only be used on files instrumented by Wallaby and when deciding what files to instrument the tool respects Jest's `collectCoverageFrom` setting. In your case the setting only lists `src/` files, and the function that you are trying to step into is defined in a file under `test/` dir, so Wallaby doesn't step into it.

**Comment by damms005** [comment]
@ArtemGovorov Thank you! By the way, I believe this is worth mentioning in the docs, or more appropriately in the [troubleshooting section](https://wallabyjs.com/docs/intro/troubleshooting.html) (which is where I went to first before opening this issue)

**Comment by ArtemGovorov** [comment]
Thanks! We'll mention it in the docs.
